doc/source/pandas_supported.rst

```diff
@@ -0,0 +1,743 @@
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_period``             | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_pickle``             | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_records``            | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_sparse``             | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_sql``                | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_stata``              | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_string``             | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_timestamp``          | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_xarray``             | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``transform``             | Y                  |                                                    |
++---------------------------+--------------------+----------------------------------------------------+
+| ``transpose``             | Y                  |                                                    |
++---------------------------+--------------------+----------------------------------------------------+
+| ``truediv``               | Y                  | See ``__truediv__``                                |
+|                           |                    |                                                    |
+|                           |                    | ``level`` parameter not yet supported              |
++---------------------------+--------------------+----------------------------------------------------+
+| ``truncate``              | N                  | N/A, Not Yet Implemented   
```

Comment:
Explain what it means here to default to pandas. ie. convert this to a non-distributed pandas DataFrame and use the underlying IO.